devtools/mkrelease

```diff
@@ -1,41 +1,16 @@
-#!/usr/bin/env python
-
-import os
-import sys
-import subprocess
-
-python = sys.prefix + '/bin/python'
-
-cwd = os.getcwd()
-
-try:
-    sys.argv.remove('--amber')
-    ambertools_distro = True
-except ValueError:
-    ambertools_distro = False
-
-pytraj_dir = os.path.abspath(os.path.dirname(__file__) + '/../')
-print('pytraj_dir', pytraj_dir)
-os.chdir(pytraj_dir)
-
-subprocess.check_call('git clean -fdx .', shell=True)
-
-# change to release status
-with open('scripts/base_setup.py', 'r') as fh:
-    old_content = fh.read()
-    new_content = old_content.replace('is_released = False',
-                                      'is_released = True')
-
-with open('scripts/tmp.py', 'w') as fh:
-    fh.write(new_content)
-
-subprocess.check_call('mv scripts/tmp.py scripts/base_setup.py', shell=True)
-subprocess.check_call('{} scripts/cythonize.py'.format(python), shell=True)
-
-if not ambertools_distro:
```

Comment:
Please keep this logic.
It will be used in ambertools git.
Basically, if `pytraj` is released in ambertools, we will only generate cythonized files and skip making tar file.